dockerfiles/hpcc/values.yaml

```diff
@@ -59,7 +59,7 @@ xroxie:
       Roxie:
         useAeron: true
 
-thor:
+xthor:
```

Comment:
deliberate?

Reply:
No, and I removed. Though the temptation to disable thor will decrease once you get on-demand working...